composer.json

```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=7.2.0"
```

Comment:
We'll just need to make sure we do a major version update for this repository if we're updating the PHP requirement